### ISSUE TITLE
It is too hard to follow the competition from outside

### ISSUE THREAD

**Issue by semeyerz** [RoboCupAtHome/RuleBook#357]
Even for experienced RoboCupers it is really hard to follow the competition from outside.
This is a problem for the public image of the league but one of my biggest concerns in this regard is the attraction of new teams.
If nobody is able to follow the competition and get a feel for it they will most likely not participate in the next year.

There are multiple resources for information on the league but none of them aggregates them.
- http://www.robocupathome.org/ : no information at all on the ongoing competition, not even links
- https://github.com/RoboCupAtHome/Nagoya2017 : You can see the tasks being prepared and so on
- https://robocup2017.org/eng/competition_schedule.html : here you can find a word document with the schedule 
- https://robocup2017.org/eng/results_athome.html : The results that most of the time lack behind one day
- Social Media: You can try to find some teams or individual TCs posting results
- https://www.youtube.com/channel/UCxbXbNOZT7BNLQNNNAU6e_w : after you found the URL of the livestream you will almost never see @home on it

So none of theses really provide information on the ongoing competition or the detailed results.

Another big part of the competition are the teamleader meetings in the afternoon.
A lot of discussion and decisions take please in these meetings.
Unfortunately there is no access to proceedings from these.

**Comment by LoyVanBeek** [comment]
+1, I totally agree. 

The league is currently not very attractive to see and hard to know what is going on. With Soccer, it's quite obvious and easy to explain. 

Maybe one committee member should have the task of doing basically such kind of public relations. It doesn't have to take much time, but is very valueable.

**Comment by kyordhel** [comment]
Considering the amount of work involved, this work can't be simply assigned. 

OC is awaiting volunteers for the task.

**Comment by LoyVanBeek** [comment]
The task can be distributed, but someone needs to do the distribution :-). 
I could have doen some things for this, but I did not for a second think about our website during the event, only focused on there and then.

**Comment by kyordhel** [comment]
This distribution needs to be done before the competition. Once the competition starts, we barely have time for thinking about who can be watching or following (I tried to publish updates I. Facebook but there was few to publish and lots f pending work). 

For instance, in Nagoya we delivered he scoring only at night because we didn't had time to capture the scores and get the signatures. If we don't have enough people to make the competition run smooth as a Swiss clock, all extra stuff is a luxury we can't afford.

**Comment by warp1337** [comment]
Bump. 

Maybe we can at least brainstorm some approaches on how to 'advertise' the @home league some more and make it more transparent for visitors/watchers. So, if we look at the 'outside world', let's e.g. consider the e-sport community, it's all about streaming the event and having it commented (at least to some degree). 

I KNOW, we are already lacking people helping us to RUN the league at the event itself. However, c'mon people it's roughly 9 Months until RC 2018. 

I would like to discuss the following:

Who could comment on the tasks? Maybe some of the famous @home people like Tijn van der Zant, @dirkholz (if he's not too busy).
Streaming: well, that's almost easy to solve, look at https://www.twitch.tv/ for example. Almost everybody is using it nowadays.

**Comment by balkce** [comment]
The issue is not necessarily on which tool to stream. YouTube Live Streaming was carried out in 2017: https://www.youtube.com/channel/UCxbXbNOZT7BNLQNNNAU6e_w By the way, I believe this is more suitable than Twitch, which is more for gaming.

The issue is more of who puts this together before and during the competition. You seem to be very interested in this topic: care to be our Social Media Tsar?

**Comment by warp1337** [comment]
Hmmm, I guess I have a team to lead (ToBI). What I can do is to help organizing this effort and probably make some advertisement before the world cup. 

You are right, it's not so much about the platform, BUT it ALL about comments. Nobody, really nobody gets the @home league without comments. I've been in the @home league since 2010 and I would never watch a live stream without a comment and a moving camera.

**Comment by justinhart** [comment]
I don't think that concentrating on who is announcing the match is the most
relevant thing here. People crowd around the arena and can't see. They're
not trying to hear someone explain what is going on, they're trying to see
the robot doing things for themselves. If you want to make this experience
better for the viewer, you need to focus on people being able to see the
robot for themselves. This means bleachers for the audience, so they're not
all crowding next to the arena, good camerawork, and an overhead monitor.

In terms of streaming, my friends and family really couldn't find the @home
stream last year. They were tuning into the RoboCup channel and always
found soccer. Perhaps a broadcast grid saying what channel to see what
match at what time is in order, and then careful curation of the footage
after. I can't find my own team's videos from last year (if they exist).

This isn't to sound critical. I'm just pointing out how the previous
successful examples of this work. If you want more exposure, *this* is
where you focus your efforts. The e-sports example is a good one. I've been
to these matches. There's a giant screen in front of an auditorium with a
big audience. The streams are immediately archived so people can watch the
matches that interest them at home.

**Comment by warp1337** [comment]
@justinhart Hi there, I hope you're well! I thanks for joining this discussion.

I am applying a filter now:
We have to issues that are related and one that's trivial to solve.

**a)** We want more people watching and following the @home league remotely, e.g, via network stream (platform doesn't seem to be a big problem, there are well working solutions out there. YT, twitch, you name it)
**b)** Justin also brought up the user experience watching the tests/contest **on location** and IMHO, he is right. 
**c)** A landing page for @home streams

What we need for both a) and b) is: "good camerawork, and an overhead monitor", I totally agree.
I also agree that on location we probably don't need any comments because the "live experience" might just be good enough. Also, the people on location may ask bystanders "what is actually happening here?" 

In terms of a) I must insist on a commented stream. As I said earlier, if we want to gain attraction someone needs to explain to the **remote** watchers what is actually happening, and to be honest, most of the times what is actually NOT happening and why that might be or why the current situation is difficult for robots. Also, I agree that it doesn't matter who comments on the video, but a comment in general is crucial for the user-experience. The target audience maybe never had any contact with the @home league before. You simply don't get @home just by watching a robot driving slowly through an artificial apartment.

c) That's just a technicality from my point of view. Should be easy to setup.

**Comment by LoyVanBeek** [comment]
Gathering from this, I think the LOC should have a few people responsible for (social) media. 

Perhaps the vizbox can help explaining what is happening to both the local & remote audience. 
But i totally agree that explaining what is **not** happening and why something is difficult is also very important. Most tasks can be achieved just as well by a toddler and much faster even. 

Perhaps the vizbox can help in that regard, but it's not a solution to all problems. 
The problem with live commentary, which we have had for @Home occasionally, is that the speech (might) interfere with the robot. The commentator has to know when to stop talking.

**Comment by justinhart** [comment]
Yes, Loy brings up a significant point here. I don't know that it's as
simple as the commentator knowing when to stop talking. Most of the tasks
involve operators verbally instructing the robot. I know that there is a
line in the rulebook stating that the robot should be able to handle
interference from sounds in the outside the arena, including the
commentator, but I think that part of our job should be to assure that the
sound that the robots receive is relatively clutter-free.

I see the "for the commentator" perspective, but I am still not sold that a
commentator is the way to accomplish audience engagement. The tasks in
RoboCup@Home seem simple enough that they are understandable without an
announcer explaining it. The sport of golf, while it has commentators, has
absolute silence while the golfer is actually taking a shot. If the verbal
instruction to the robot is, "Pick up the cup and place it in the cabinet,"
doesn't the commentator just say, "The operator asked the robot to pick up
the cup and place it into the cabinet?"

**Comment by LoyVanBeek** [comment]
Indeed, that would be quite pointless. The [vixbox](https://github.com/LoyVanBeek/vizbox) already shows the conversation as well. 
I think the comments should be more focusing on why technicalities involved ("it's using a ... 3D camera with a limited field of view, like seeing through a roll of toilet paper so it's very difficult to see much. The images feed into a neural net ..."). 
But also why things are relevant for the audience in their homes later on. 

This is most interesting to tell when the robot is not doing what it's supposed to, so we need a knowledgeable commentator (e.g. most likely someone talking in this issue). 

Alternatively, we can show the deeper explanation text in the vizbox as well, alternating this text with camera images for example.

**Comment by balkce** [comment]
By the way, does anybody know who is the LOC?

I think they should be part of this discussion.

For example, the URL [http://robocup2018.org](http://robocup2018.org) hasn't even been bought yet. And the Facebook page https://www.facebook.com/RoboCup2018/ has the dates of the competition wrong.

**Comment by LoyVanBeek** [comment]
@maximest-pierre is part of it at least IIRC.

**Comment by robotijn** [comment]
Hi,

I think the problem is more profound...
The tests are too complex. There are too many details.
I'm not saying the tasks within the tests are too complex. But the
implementation of the tests.

Why not simplify the tests?

For example,  the follow me in Mexico did very well. Go to the elevator,
 take the elevator to another floor and go to a person standing behind a
small crowd.
Easy to understand.

I think every test should be explainable in 1 or 2 sentences. Including the
scoring. Then we can display the exoneration of the test in several
languages and also put it on our homepage.

Cheers,

Tijn

Op 26 sep. 2017 21:37 schreef "Loy" <notifications@github.com>:

**Comment by LoyVanBeek** [comment]
This comes up every once in a while. We don't have to give out all the technicalities (that make it fair, make it judge-able, prevent simple cheating, make it practical in a busy competition etc.) to the audience. 
AFAIK, all challenges can be explained in a 1-2 sentences.

Soccer also seems simple (get the ball in the white rectangle thingy thing but there's 11 other dudes in the way somehow), but don't ask about "offside" (or "buitenspel" in Dutch). Then there's a lot more rules under the simple looking surface, apparently needed to keep it fair etc. 
I don't know all the rules and I don't need those to watch the occasional soccer match I happen to stumble into.

**Comment by warp1337** [comment]
Actually, I talked to a few former team members of ToBI. In principle, they are willing to comment a stream. Note: this is no **_promise_** or carved in stone. If this undertaking becomes more clear, I will
add them to the discussion. There are, however, lots of preconditions to resolve first.

**Comment by johaq** [comment]
I am the one interested but as @warp1337 said it is very difficult to make any commitment so far in advance.

I think the most important thing a commentator can provide I see not discussed here. Take german commentators of american football, a fairly niche sport here, as an example. While they do explain the rules of the game they mostly provide the stories that german audiences are not familiar with. E.g. these two teams have a long standing rivalry, this player was booted from the team he is now playing against etc.
The same can be applied to @home. These two teams are battling it out for first place. This long participating team has a chance to make the final for the first time and so on. And I agree that simplified tasks are desirable but without knowledge about the teams, the current standings and some history it will never be very interesting to watch 30 robots in a row do more or less the same thing.

**Comment by nickswalker** [comment]
^ I too think this is critical. Some people are compelled by stats and explanation, but good stories can help engage a broader audience.

**Comment by kyordhel** [comment]
Abandoned.
